builds/buildkite_agent_hook.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# This is a BuildKite Agent hook, which gets downloaded from S3 onto the
+# running machine, and is run before the start of every job in the pipeline.
+# See https://buildkite.com/docs/agent/v3/hooks
+
+set -o errexit
+set -o nounset
+
+echo "Hello! I am the agent hook downloaded from S3!"
+
+echo "These are the containers which are running, probably from a previous job:"
+docker ps
+
+echo "I'm going to stop these containers now, so this job has a clean start."
+docker stop $(docker ps -q) || true
```

Comment:
Given that we are not bothered about stopping containers gracefully, could this be `docker kill`?

Reply:
Sounds good. I hadn't heard of the command before; will change.